packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/UpgradeGuides.f/v7.0.0.s.tool.mdx

```diff
@@ -0,0 +1,150 @@
+import { Notice } from '@cerner/terra-docs';
+
+# v7.0.0 Upgrade Guide
+
+This document will provide information on upgrading from `terra-dev-site` `6.x.x` to `7.0.0`.
+
+`terra-dev-site` v7 is about 70% new. The primary goals of this upgrade is to enable webpack 5, prepare for `terra-application` v2, and remove the `terra-dev-site` build intermediate step by converting to a pure webpack plugin.
+
+## Scoped package
+
+Terra dev site has changed to be scoped under the cerner organization. The package has gone from `terra-dev-site` to `@cerner/terra-dev-site`;
+
+```diff
+{
+ "dependencies": {
+-  "terra-dev-site": "^6.0.0",
++  "@cerner/terra-dev-site": "^7.0.0",
+ }
+}
+```
+
+## Webpack Plugin
+
+In `terra-dev-site` v6 we introduced the webpack plugin as an alternative to the shared webpack config. In `terra-dev-site` v7 we have removed the shared webpack config and are only offering the webpack plugin. We've done this to facilitate the transition from `terra-toolkit` webpack config to the new `@cerner/webpack-config-terra` webpack config. `terra-dev-site` will work with either webpack config and this will allow you to update to WDIO 6 independently.
+
+To simplify the webpack plugin api we have also now included the Directory Switcher and Local Package alias resolve plugins in the `terra-dev-site` plugin by default. As a reminder, these plugins allow you to enable hot reloading by switching lib requires with their src equivalent and reference local files as if they were packaged, ie `import('@cerner/terra-dev-site/file)`.
+
+In addition the `TerraDevSiteEntrypoints` have been removed as well. Webpack 4 requires an entry point, but that requirement is generally satisfied by our reusable webpack configs. Webpack 5 will allow plugins to define entrypoints without being specified in the config.
+
+
+Below is an example webpack config setting up the `terra-dev-site` plugin.
```

Comment:
This is for v7 correct? Would a diff of v6 to v7 be helpful for migration?

Reply:
I think the diff would just end up looking messy. See https://github.com/cerner/terra-core/pull/3271/files for an example.

Reply:
ahh it is a bit messy. Might be worth linking to that commit for teams to reference that?